wondrous-app/components/SingleDatePicker/SingleDatePicker.tsx

```diff
@@ -0,0 +1,155 @@
+import { useMemo, useState } from 'react';
+import { DayPickerSingleDateController } from 'react-dates';
+import moment from 'moment';
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+
+import {
+  DATEPICKER_OPTIONS,
+  DEFAULT_SINGLE_DATEPICKER_VALUE,
+  WEEK_DAYS,
+  DAY_STRING_MONTH_SHORT_YEAR,
+  MONTH_DAY_FULL_YEAR,
+} from 'utils/constants';
+
+import DatePickerRecurringUtilities from 'components/DatePickerRecurringUtilities';
+import DatePickerNavButton from 'components/DatePickerNavButton';
+import CalendarDay from 'components/CalendarDay';
+
+import styles from './SingleDatePickerStyles';
+
+const SingleDatePicker = ({ sx }) => {
```

Comment:
i think we need to take in values from props so that this component can have initial values, for example when we are editing task, we want to prepopulate this component with existing dates and repeatingValue.